rest/permissions.py

```diff
@@ -105,7 +106,7 @@ def has_permission(self, request, view):
         except PermissionDenied:
             return False
 
-        if request.method in SAFE_METHODS:
+        if request.method in self.SAFE_METHODS_CUSTOM:
```

Comment:
Actually, I take back the last comment, since the panel should not be available if the user does not have permissions to edit the taxonomy (and if it does not work in this way, we have a bug)